### DISCUSSION TITLE
Creating user-inface using Flood Modeller api

### DISCUSSION THREAD

**Discussion by murtazasarwar** [People-Places-Solutions/floodmodeller-api#17] [Show and tell]
Using QGIS as the vehicle, see the user-intreface created using the functionalities provided by the Flood Modeller api.

https://www.linkedin.com/posts/murtaza-sarwar-472887a_floodmodeller-activity-6957015932548325376-yRxz?utm_source=linkedin_share&utm_medium=member_desktop_web

**Comment by joe-pierce** [comment]
This is a nice idea! Would be a handy way to roll out tools to non-python users.
